ext/src/content/content.js

```diff
@@ -417,42 +221,242 @@ function init() {
       });
       const newCommentObserver = new MutationObserver(() => {
         commentObserver.disconnect();
-        setupCommentObserver(commentObserver);
+        resetGithubCommentObserver(commentObserver);
       });
       const pageObserver = new MutationObserver(() => {
         if (window.location.pathname !== currentPath) {
           currentPath = window.location.pathname;
           initGitHubButton();
 
           commentObserver.disconnect();
-          setupCommentObserver(commentObserver);
-          const content = document.querySelector(".js-disscussion");
-          if (content) {
-            newCommentObserver.disconnect();
-            newCommentObserver.observe(content, { childList: true });
-          }
+          resetGithubCommentObserver(commentObserver);
+          resetGithubNewCommentObserver(newCommentObserver);
         }
       });
       pageObserver.observe(document.querySelector("body"), {
         childList: true
       });
-      const jsDiscussion = document.querySelector(".js-discussion");
-      if (jsDiscussion) {
-        newCommentObserver.observe(jsDiscussion, {
-          childList: true
-        });
-      }
-      setupCommentObserver(commentObserver);
+      resetGithubCommentObserver(commentObserver);
+      resetGithubNewCommentObserver(newCommentObserver);
       isGithubListenerAdded = true;
     }
     initGitHubButton();
   }
 
+  // Pasting events are used to force stackoverflows WMD to save state.
+  // TODO: This solution is buggy and does not always work - more discussion needed on solution
+  function stackOverflowFormat(event, inputEl) {
+    event.preventDefault();
+    const codeBlocks = getCodeBlocksStackOverflow(inputEl);
+
+    let markdownText = inputEl.value;
+    const formattedText = [];
+    if (codeBlocks.length) {
+      /*
+       * TODO: Add support for language-all: <!-- language-all: lang-* -->
```

Comment:
Any idea where [this](https://github.com/prettier/prettier-chrome-extension/commit/0407f000ce6632eb2fc54adb7b2672d8efafc0b6) went? Looks like a bad merge conflict resolution.